### PR TITLE
Fix native issues in new functionality

### PR DESCRIPTION
**Pull request by KseniaBuntova**
## Technical description

- [x] Feature description...

## Tested on

-
- 

## Steps to reproduce

1. 
2. 

## Screenshots (optional):

No screenshots.

*This Pull Request template has been written and generated by Monk JS repository.*

### CODE REVIEW

File: packages/inspection-report/src/components/CarView360/svgElementMapper/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /* eslint-disable react/no-array-index-key */
 import PropTypes from 'prop-types';
 import React, { useMemo } from 'react';
-import { CommonPropTypes } from '../../resources';
+import { CommonPropTypes } from '../../../resources';
```
Comment (artgoncharov) [inline packages/inspection-report/src/components/CarView360/svgElementMapper/index.js:4]: is it valid? should it be only native changes?

File: packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js [hunk 2]
```diff
@@ -41,7 +39,7 @@ const styles = StyleSheet.create({
     paddingBottom: 6,
     paddingLeft: 16,
     paddingRight: 16,
-    top: 650,
+    top: 200,
```
Comment (artgoncharov) [inline packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js]: does it affect web?

File: packages/inspection-report/src/components/CarView360/svgElementMapper/index.js (hunk 1 above)
Comment (KseniaBuntova) [reply packages/inspection-report/src/components/CarView360/svgElementMapper/index.js:4]: Yeah due to moving this file in folder this is a necessary update. And moving it to folder is required by linter rules cause I added native realisation for this file

**Review by knoskov — APPROVED** [review]

File: packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js [hunk 3]
```diff
@@ -157,10 +161,12 @@ export default function UpdateDamagePopUp({
         if (gestureStat.moveY <= topLimitY) {
           pan.setValue({ x: 0, y: topLimitY });
         } else if (gestureStat.moveY >= bottomLimitY) {
-          pan.setValue({ x: 0, y: bottomLimitY });
+          pan.setValue({ x: 0, y: pan.y });
```
Comment (artgoncharov) [inline packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js]: it's changing web behaviour?

**Review by artgoncharov — APPROVED** [review]

**Pull request merged**